src/features/Layout/Layout.jsx

```diff
@@ -21,28 +21,33 @@ import SignupForm from '../Auth/SignupForm';
 import ModalWithConfirmationBox from '../../common/ModalWIthConfirmation/ModalWithConfirmationBox';
 import { logout } from '../../utils/utils';
 import { authenticationAPI } from '../../services/authentication';
+import { useGetUserQuery, userAPI } from '../../services/user';
 
 export default function Layout() {
   const [signOutMutation] = authenticationAPI.useGetSignOutMutation();
-  const [user, setUser] = useState(localStorage.getItem('userID'));
+  const { data: user, refetch } = useGetUserQuery();
+  const [changeTheme] = userAPI.useChangeThemeMutation();
 
   const [openDrawer, setOpenDrawer] = useState(false);
-  const [isLightMode, setIsLightMode] = useState(true);
   const [showSignupModal, setShowSignUpModal] = useState(false);
 
   const handleDrawerOpen = () => setOpenDrawer(true);
   const handleDrawerClose = () => setOpenDrawer(false);
 
-  const handleLoginSuccess = (data) => {
-    setUser(data.id);
+  const handleLoginSuccess = async () => {
+    await refetch();
```

Comment:
https://redux-toolkit.js.org/rtk-query/api/created-api/hooks#uselazyquery 

should be used instead of async await.